src/components/Footer/Footer.js

```diff
@@ -188,6 +282,7 @@ const Footer = ({
           </TouchableOpacity>
         </View>
       </View>
+      <BottomSheetMenu height={186} refSheet={refSheet} dataSheet={dataSheet} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The BottomSheetMenu component has been added to the render method. It uses the `refSheet` ref and the `dataSheet` array defined earlier. No issues found here.
```diff
+ <BottomSheetMenu height={186} refSheet={refSheet} dataSheet={dataSheet} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->